packages/zowe-explorer/src/extension.ts

```diff
@@ -49,6 +50,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
     registerRefreshCommand(context, activate, deactivate);
     initializeSpoolProvider(context);
 
+    PollDecorator.register();
+
```

Comment:
Since the PollDecorator is generalized for any polling operation, I've kept it separate from the `initializeSpoolProvider` logic.

I think as long as `PollDecorator.register` is called after the relevant schemes are registered in VScode, then we (hopefully) will avoid any issues w/ registering the `FileDecorator` 😄